pkg/hostpath/hostpath.go

```diff
@@ -191,15 +190,15 @@ func commonUpsertRanger(obj runtime.Object, podPath, filter string, key, value i
 	// on an update we first nuke any existing directory and then recreate it to simplify handling the case where
 	// the keys in the secret/configmap have changed such that some keys have been removed, which would translate
 	// in files having to be removed
-	if err := os.RemoveAll(podFileDir); err != nil {
+	if err := commonOSRemove(podPath); err != nil {
```

Comment:
nit - shouldn't the comment above be updated, since we are no longer isolating content in the type/namespace:name subdirectory?

Reply:
yes it should

will update that and push in a few

Reply:
comment updated @adambkaplan